ClosedXML/Excel/Ranges/XLRangeBase.cs

```diff
@@ -841,35 +803,28 @@ public XLCell Cell(XLAddress cellAddressInRange)
             if (cell != null)
                 return cell;
 
-            Int32 styleId = GetStyleId();
-            Int32 worksheetStyleId = Worksheet.GetStyleId();
+            var styleValue = this.StyleValue;
 
-            if (styleId == worksheetStyleId)
+            if (styleValue == Worksheet.StyleValue)
             {
                 XLRow row;
                 XLColumn column;
                 if (Worksheet.Internals.RowsCollection.TryGetValue(absRow, out row)
-                    && row.GetStyleId() != worksheetStyleId)
-                    styleId = row.GetStyleId();
+                    && row.StyleValue != Worksheet.StyleValue)
```

Comment:
`.Equals()` ?

Reply:
> Maybe XLStyleValue should override == and != ?

I was pretty sure they are implemented ) Will do right away.

Reply:
As long as styles are shared it is OK to rely on reference comparison, but anyway, to have explicit operators will be better.